### PR TITLE
Client has a hero component 

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[ PT Story](https://www.pivotaltracker.com/story/show/178433044)

## User story:
```
As a visitor
In order to get a good first impression about the business
I'd like an appealing landing page
```

## Changes proposed:
- Adds and styles landing page components

## Lessons learned:
- Nesting with sass
- Customizing material ui a bit more

## Dependencies
- React router

## Screenshots
image

https://recordit.co/sK4feKGmdb

### CODE REVIEW

**Review by 1CIM — CHANGES_REQUESTED** [review]
Looks nice boiis!
But how dose it look at iPhone x size (375 width)? 
phone number not so responsive right 🥲
pls fix <333

File: src/custom/CustomButton.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      data-cy={dataCy}
+      onClick={() => clickHandler}>
+      {children}
+    </button>
+  )
+
+  return <>{link ? <Link to={to}>{button}</Link> : button}</>
+}
+
+export default CustomButton
```
Comment (offcast-dev) [inline src/custom/CustomButton.jsx:24]: damn! I don't understand the code. What is children for?

**Review by offcast-dev — APPROVED** [review]

File: cypress/integration/userCanVisitLandingPage.feature.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+describe('User can visit Flex Coast landing page', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('shows the main landing page', () => {
+    cy.get('.hero-section')
+      .should('have.css', 'background-image')
```
Comment (DavDan1) [inline cypress/integration/userCanVisitLandingPage.feature.js:8]: this looks crazy nice!

**Review by DavDan1 — APPROVED** [review]

File: src/custom/CustomButton.jsx (hunk 1 above)
Comment (LucasKnudsen) [reply src/custom/CustomButton.jsx:24]: Everything inside the CustomButton is sent down as props within the key `children` :D
```
<CustomButton>
  // children goes here
</CustomButton>
```

**Review by 1CIM — APPROVED** [review]

**Pull request merged**